src/Arachne.cc

```diff
@@ -220,6 +171,13 @@ bool useCoreArbiter = true;
 
 CoreArbiterClient* coreArbiter = NULL;
 
+/*
+ *  The CorePolicy that Arachne will use.  It is loaded in the init
+ *  function.
```

Comment:
I think you can skip the second sentence. If people really care about this, they can search for uses of the name (don't want to tie down the implementation unnecessarily).